pom.xml

```diff
@@ -8,7 +8,7 @@
     <artifactId>declarativefx</artifactId>
     <name>DeclarativeFX</name>
     <url>https://github.com/grevend/DeclarativeFX</url>
-    <version>0.5.3-SNAPSHOT</version>
+    <version>0.5.4-SNAPSHOT</version>
```

Comment:
0.5.4??? you mean 0.5.5?

Reply:
The changes for 0.5.5 are still pending.